ROMFS/px4fmu_common/init.d/rc.io

```diff
@@ -1,17 +1,93 @@
+#!nsh
 #
-# Init PX4IO interface
+# Script to configure control interface
+#
+#
+# NOTE: environment variable references:
+#  If the dollar sign ('$') is followed by a left bracket ('{') then the
+#  variable name is terminated with the right bracket character ('}').
+#  Otherwise, the variable name goes to the end of the argument.
 #
 
 #
-# Allow PX4IO to recover from midair restarts.
+# Declare local script variables and initialize default values.
 #
-px4io recovery
+
+set IO_FILE             /etc/extras/
 
 #
-# Adjust PX4IO update rate limit
+# Set USE_IO flag.
 #
-set PX4IO_LIMIT 400
+if param compare SYS_USE_IO 1
+then
+	set USE_IO yes
```

Comment:
I think we have an ordering problem here: rc.interface sets USE_IO, but rc.io is executed first.
IO_RESENT needs inspection as well.

Reply:
Again, thanks for your review @bkueng.  USE_IO is now declared at the top level rcS to allow airframe config files to set/manipulate that parameter for lower level scripts to use as they currently do.  This appears to have silenced one warning that exists in PX4/Firmware master `ERROR [param] Parameter SYS_USE_IO not found`.  

IO_PRESENT logic has been repaired as well.